src/TourCMS.php

```diff
@@ -172,6 +173,34 @@ public function set_user_agent(string $user_agent, bool $prepend = true) {
 		return true;
 	}
 
+	/**
+	 * add_header
+	 *
+	 * @author Francisco Martínez Ramos
+	 * @param string $header Key of the header
+	 * @param string $value Value of the header
+	 * @return bool
+	 */
+	public function add_header(string $header, string $value): bool
+	{
+		$new_header = "$header: $value";
+		array_push($this->headers, $new_header);
+		return true;
+	}
+   
+	/**
+	 * set_request_identifier
+	 *
+	 * @author Francisco Martínez Ramos
+	 * @param string $value Value of the request identifier header
+	 * @return bool
+	 */
+	public function set_request_identifier(string $value): bool
+	{
+		$value = $value . " (" . uniqid() . ")";
```

Comment:
I'm not sure we should prepend our own unique id, we should let the implementer define that.

```suggestion
```